.changeset/great-mangos-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vercel/next': patch
+'vercel': patch
```

Comment:
Changesets should handle bumping `vercel` if any of the dependencies are bumped, so this should be redundant! 😄